### ISSUE TITLE
Fix Depths

### ISSUE THREAD

**Issue by JessicaShippen** [planetirf/math456_soil_project#6]
Make a consistent depth system.

**Comment by planetirf** [comment]
The correct depth classes should be:

0-12
12-24
24-36
36-48 (if it exists)

********
0-8  is also valid but did not follow the correct sampling scheme.

**Comment by recursory** [comment]
This issue was at least partially resolved by refactoring depth-1 into depth-low and depth-high, right? If something else needs to be done, could we close this and make it a separate issue?

**Comment by planetirf** [comment]
The mis-matching depths issue was solved by using the new dataset PCP-R-ready.csv, that was sent from an old student I reached out to last week. 

The total observations were reduced from ~1200 to ~850 in the new data set. It also applied a sampling date and an irrigation variable which was not available in the previous set.  

The splitting of the columns from  a single factor to a top and bottom is sort of a data quality practice and the correct format for the soil-data R packages i'm learning which might be beyond the scope of this class.

The script runs well on this data set and there are no mis-labled  values in the new dataset.

[CLOSED]
